py/vttest/local_database.py

```diff
@@ -50,9 +54,14 @@ def vtgate_addr(self):
 
   def config(self):
     """Returns a dict with enough information to be able to connect."""
-    return {
-        'port': vt_processes.vtgate_process.port,
-        }
+    if self.mysql_only:
+      return {
+          'socket': self.mysql_db.unix_socket(),
```

Comment:
Yeah. My plan is to try importing this, and fixing forward on the necessary changes. This is just to get the initial test working.